src/documents/AchievingInclusiveEducation.md

```diff
@@ -37,7 +48,8 @@ Learning needs that affect learning can include:
 <figure>
 [![Floe concept design mindmap](/assets/images/thumbs/400px-Floe_concept_design_mindmap.png)](/assets/images/Floe_concept_design_mindmap.png)
 <figcaption>
-*Figure 1:* Floe concept design mindmap illustrates the many different ideas and parts which work together to enable a one-size-fits-one education.
+*Figure 1:* Floe concept design mindmap illustrates the many different ideas and parts which work together to enable a
```

Comment:
An unecessary `<br/>` appears at the end of this line when rendered, causing the next line to always be on a separate line.

![image](https://user-images.githubusercontent.com/564762/98133143-3bf2fe00-1e8b-11eb-83fa-20291b668942.png)

Reply:
Good catch. I guess we need to exempt markdown within figures from the line length rule?

Reply:
Yes I think that makes sense.

Reply:
I'm not sure what the issue is exactly, but depending on the parser you may not be able to mix the markdown inside of the HTML.

Reply:
@jobara See notes about figures here: https://github.com/fluid-project/docs-inclusive-learning/pull/29